### PR TITLE
Prepare version 4.1

### PR DESCRIPTION
**Pull request by OsirisTerje**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -11,17 +11,17 @@ Requests for new features and bug reports keep the project moving forward.
 ### Before you submit an issue
 
 - Ensure you are running the [latest version](https://github.com/nunit/nunit/releases) of NUnit.
-- **Many 3rd party test runners do not support NUnit 3 yet.** They may appear to run NUnit tests, but they do not do so correctly.
-- To ensure that the bug is in NUnit and not in the runner you are using, **test your bug using the nunit3-console.exe** first.
+- **Many 3rd party test runners do not support NUnit 3/4 yet.** They may appear to run NUnit tests, but they do not do so correctly.
+- To ensure that the bug is in NUnit and not in the runner you are using, **test your bug using dotnet test, nunit3-console or nunitlite** first.
```
Comment (SeanKilleen) [inline CONTRIBUTING.md:15]:
```suggestion
- To ensure that the bug is in NUnit and not in the runner you are using, **test your bug using `dotnet test`, `nunit3-console` or `nunitlite`** first.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -11,17 +11,17 @@ Requests for new features and bug reports keep the project moving forward.
...
 - **Search** the [issue list](https://github.com/nunit/nunit/issues?utf8=%E2%9C%93&q=is%3Aissue) (including closed issues) to make sure it hasn't already been reported.
 
 ### Submitting a good issue
 
 - Give the issue a short, clear title that describes the bug or feature request
-- Include what version of NUnit you are using
-- Tell us how you are running your tests including command line arguments for the console runner
+- Include what version of NUnit you are using, which runner you are using
+- Tell us how you are running your tests including command line arguments or runsettings
```
Comment (SeanKilleen) [inline CONTRIBUTING.md:22]:
```suggestion
- Tell us how you are running your tests including command line arguments or `runsettings`
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -38,14 +38,16 @@ If you’re getting ready to report a bug in NUnit, figuring out how to create a
 
 ## Documentation
 
-Great documentation is essential for any open source project and NUnit is no exception. [Our documentation](https://docs.nunit.org/articles/nunit/intro.html) often lags behind the features that have been implemented or would benefit from better examples.
+Great documentation is essential for any open source project and NUnit is no exception. 
 
-A great place to start is by going through the [NUnit release notes](https://github.com/nunit/docs/wiki/Release-Notes) and ensuring that the documentation for new features is up to date.
+[Our documentation](https://docs.nunit.org/articles/nunit/intro.html) often lags behind the features that have been implemented or would benefit from better examples.
```
Comment (SeanKilleen) [inline CONTRIBUTING.md:43]: I think this should be an xref to the article id. See other examples throughout docs, or I can fix it up later today before merge

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -38,14 +38,16 @@ If you’re getting ready to report a bug in NUnit, figuring out how to create a
...
 
 ## Fixing Bugs and Adding Features 
 
 We love pull requests, but would prefer that new contributors start with smaller issues and let us know before you contribute to prevent duplication of work.
 
+Ensure you do have an issue to connect your pull request to. If there isn't one, please open a new issue first, and state you have a fix before embarking on the pull request.
```
Comment (SeanKilleen) [inline CONTRIBUTING.md:49]:
```suggestion
Ensure you have an issue to connect your pull request to. If there isn't one, please open a new issue first, and state you have a fix before embarking on the pull request. 
```

**Review by SeanKilleen — COMMENTED** [review]
LGTM with a few minor suggestions

**Comment by OsirisTerje** [comment]
I'm trying to get this out before anyone else add in a new PR.  The contributing  doc is not that important, and been like that since forever.  Can we(you) fix this up post release ?

File: CONTRIBUTING.md (hunk 3 above)
Comment (OsirisTerje) [reply CONTRIBUTING.md:43]: But will xref work from the ordinary git repo to the docfx docs ?

**Pull request merged**